docs/src/content/docs/de/reference/plugins.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Plugins Referenz
+description: Ein Überblick über die Starlight-Plugin-API.
+tableOfContents:
+  maxHeadingLevel: 4
+sidebar:
+  label: Plugins
```

Comment:
Related discussion for reference [withastro/starlight/pull/2470/files/2b486d1b1fa4acf3f0c87da750f3ce219794f78e#diff-9185fd84aba7e9a14b394be96e2bd3562baac10c0285be708a3109c5cbe72802](https://github.com/withastro/starlight/pull/2470/files/2b486d1b1fa4acf3f0c87da750f3ce219794f78e#diff-9185fd84aba7e9a14b394be96e2bd3562baac10c0285be708a3109c5cbe72802)